wordpress-mobile/WordPress-iOS
# ULS: Update Auth to use 2FA view with SMS code request

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/336
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/347

To test:
- **Enable the `unifiedGoogle` feature flag.**
- Select `Continue with Google` from either login or signup.
- Select a Google account that:
  - _Does not_ have a WP password set.
  - _Does_ have WP 2FA enabled.
- Tap `Text me a code instead`.
- Verify:
  - The HUD is displayed.
  - You receive a text message
  - You can log in with the code.

<kbd>![Simulator Screen Shot - iPhone 11 Pro Max - 2020-07-30 at 12 51 19](https://user-images.githubusercontent.com/1816888/88962294-797c3580-d263-11ea-8ae2-3b8a109e39d7.png)</kbd>

PR submission checklist:
